modules/calendar/add_edit_event.php

```diff
@@ -692,6 +709,14 @@ function setEventDate($start_date, $recurrence)
 
     // done with EVENT insert/update statements
 
+      if ($_POST['form_category'] == '2' && (!$eid)) { //adding category 3 based on category 2
+        $endtime = $cat3time;
+        $starttime = $cat3time;;
```

Comment:
There's a double semi-colon here.

Reply:
Good catch. I am sending a change to remove it.